run-dask-process.sh

```diff
@@ -103,15 +98,15 @@ function buildTcpArgs {
                 "
 
     WORKER_ARGS="--rmm-pool-size=$WORKER_RMM_POOL_SIZE
-             --local-directory=/tmp/$LOGNAME 
+             --local-directory=/tmp/$LOGNAME
              --scheduler-file=$SCHEDULER_FILE
              --memory-limit=$DASK_HOST_MEMORY_LIMIT
              --device-memory-limit=$DASK_DEVICE_MEMORY_LIMIT
             "
 
 }
 
-function buildUCXWithInfinibandArgs {
+buildUCXWithInfinibandArgs () {
 
     export UCX_MAX_RNDV_RAILS=1
     export UCX_MEMTYPE_REG_WHOLE_ALLOC_TYPES=cuda
```

Comment:
With newer versions of ucx (1.13 and above) we no longer need to export this

Reply:
Thanks, fixed in [de23228](https://github.com/rapidsai/multi-gpu-tools/pull/24/commits/de232283fa27014b41b2241f34b2575e5299204a).